site/pages/react/customization/theme.mdx

```diff
@@ -118,3 +120,53 @@ const config = createConfig(
   }
 );
 ```
+
+## Custom authentication component
+
+The auth modal, by default, shows a `Sign in` text header. This header is fully cusomizable, allowing you to change the text or add an icon.
+
+### Customize header text
+
+To customize the header text, you can override the `header` or `hideSignInText` options of the [`auth`](https://github.com/alchemyplatform/aa-sdk/blob/main/account-kit/react/src/types.ts) config when calling `createConfig`.
+In this example, we hide the default header text and replace it with our own.
+
+```tsx twoslash
+// @jsx: react-jsx
+import { createConfig } from "@account-kit/react";
+import { sepolia } from "@account-kit/infra";
+
+export const confg = createConfig(
+  {
+    apiKey: "YOUR_API_KEY",
```

Comment:
oh can we fix these while we're here?

should add an import of `alchemy` to the `@account-kit/ifnra` import and then change this to:
```suggestion
    transport: alchemy({ apiKey: "YOUR_KEY" })
```